core/src/main/java/uk/ac/bbsrc/tgac/miso/core/util/PaginationFilter.java

```diff
@@ -204,6 +204,8 @@ public static PaginationFilter kitType(KitType type) {
           return institute(parts[1]);
         case "box":
           return box(parts[1]);
+        case "sequencingParameters":
```

Comment:
If they can only specify an ID, then it might make sense to not make this user visible.